Sources/BFloat16/BFloat16+Floating.swift

```diff
@@ -372,57 +372,51 @@ extension BFloat16: BinaryFloatingPoint {
   }
   
   @inlinable public static var exponentBitCount: Int {
-    get {
-      return 8
-    }
+    get { 8 }
```

Comment:
```suggestion
    8
```
If a property only has a getter, Swift lets you omit the `get` block entirely